recipes/sv2/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "1.4.1" %}
+{% set sha256sum = "16fdc9e59f837549f149906fe4e771da40265e52d165460ecf6ac8949df7397b" %}
+
+package:
+  name: sv2
+  version: {{ version }}
+
+source:
+  fn: sv2-{{ version }}.tar.gz
+  url: https://github.com/dantaki/SV2/archive/sv2v1.4.1.tar.gz
```

Comment:
You could use {{ version }} also in the url

Reply:
Good catch!